arch/arm/soc/st_stm32/stm32f4/Kconfig.defconfig.series

```diff
@@ -51,4 +51,19 @@ config PINMUX_STM32
 
 endif #PINMUX
 
+if USB
+
+config USB_DC_STM32
+	def_bool y
+
+config USB_DC_STM32_EP_NUM
+        int
+        default 4
+
+config USB_DC_STM32_RAM_SIZE
```

Comment:
where are these 2 config vars defined.  The defconfig should just be providing override values.

Also, add help description for someone to understand what these values should be set to.

Reply:
I understand your point about defconfig but the endpoint number and RAM size depends on SOC's family and USB controller. If we move interrupt and interrupt priority to the DT files, we should do the same with these two. Is it possible to do it in a following PR?

Reply:
Why wait for a follow up PR?

Reply:
This PR is based on an old PR from gerrit (https://gerrit.zephyrproject.org/r/#/c/10176/) which wasn't merged for minor issues. Moreover, there is a request, from the author of this PR, to upstream  the USB driver to support OTG FS controller on F4 family. He wants to develop USB-EEM function driver. As I mentioned above, I will close WIP PR #1384 and create a new PR to add support for SOCs with USB device controller. Merging this PR will allow things to move in parallel.

Reply:
@galak I added config var declaration in the usb device driver kconfig with some help description.